server.go

```diff
@@ -312,14 +312,23 @@ func (c *Client) ShutdownServer(id string) error {
 		method: http.MethodPatch,
 		body:   new(map[string]string),
 	}
+
 	err = r.execute(*c, nil)
 	if err != nil {
+		if requestError, ok := err.(*RequestError); ok {
+			if requestError.StatusCode == 500 {
```

Comment:
Should it be `requestError.StatusCode == 500 || requestError.StatusCode == 503` or `requestError.StatusCode >= 500`?

Reply:
Internal-server with status-code 500 occurs when a server does not have Storage/OS for graceful shutdown and I want only target this case. From API docs status-code 503: `The request could not be performed because there was a temporary error on our side`, which means some error on our API/infra and is not related to the resource (server).

Reply:
Alright. I see what you meant. 👯‍♂